internals/resource-managers/knative_service.go

```diff
@@ -74,11 +76,27 @@ func (k KnativeServiceManager) prepareResource(capp rcsv1alpha1.Capp, ctx contex
 	return knativeService
 }
 
+// prepareVolumes generates a list of volumes to be used in a Knative Service definition from a given Capp resource.
+func (k KnativeServiceManager) prepareVolumes(capp cappv1alpha1.Capp) []corev1.Volume {
+	volumes := make([]corev1.Volume, 0)
```

Comment:
Change the initialization of the slice, I don't think we initialize a slice like this anywhere in the code